README.md

```diff
@@ -53,46 +53,61 @@ security model][security]. These are represented by Scala traits called
 `Reader`, `Writer`, and `Master`. According to the level of access that your
 application requires, you must mix the appropriate trait(s) into your client
 instance when creating it, and configure your corresponding API keys. This is
-demonstrated in the example below.
+demonstrated in the examples below.
 
-Our recommended means of providing settings is through environment variables, to
-avoid storing credentials in source control. We intend to support a config file
-soon, but would still discourage you from using that for credentials.
+Configuration is supported by the [Typesafe config] library, offering all the
+flexibility you could wish to provide settings through a file, environment
+variables, or programmatically. We recommend environment variables for your API
+keys at very least, to avoid storing credentials in source control. To that end,
+the following will be honored by default if set:
 
-## Example
+* `KEEN_PROJECT_ID`
+* `KEEN_READ_KEY`
+* `KEEN_WRITE_KEY`
+* `KEEN_MASTER_KEY`
+
+To configure with a file, it must be on the classpath, customarily called
+`application.conf` though you may use others--see the Typesafe config
```

Comment:
I linked to the GitHub page for the project in the preceding paragraph, and directly to the Javadocs for the `Config` object in the next paragraph, that seemed like enough to me :grinning: